Microsoft.Maui-windows.slnf

```diff
@@ -27,8 +27,8 @@
       "src\\Controls\\tests\\CustomAttributes\\Controls.CustomAttributes.csproj",
       "src\\Controls\\tests\\DeviceTests\\Controls.DeviceTests.csproj",
       "src\\Controls\\tests\\Xaml.UnitTests.ExternalAssembly\\Controls.Xaml.UnitTests.ExternalAssembly.csproj",
-      "src\\Controls\\tests\\Xaml.UnitTests.InternalsVisibleAssembly\\Controls.Xaml.UnitTests.InternalsVisibleAssembly.csproj",
       "src\\Controls\\tests\\Xaml.UnitTests.InternalsHiddenAssembly\\Controls.Xaml.UnitTests.InternalsHiddenAssembly.csproj",
+      "src\\Controls\\tests\\Xaml.UnitTests.InternalsVisibleAssembly\\Controls.Xaml.UnitTests.InternalsVisibleAssembly.csproj",
```

Comment:
For some reason VS keeps wanting to move this so, I'm hoping if we check this in then it'll just be content.